app/src/debug/java/com/hedvig/app/feature/insurance/InsuranceMockActivity.kt

```diff
@@ -0,0 +1,101 @@
+package com.hedvig.app.feature.insurance
+
+import android.os.Bundle
+import androidx.appcompat.app.AppCompatActivity
+import androidx.recyclerview.widget.RecyclerView
+import com.hedvig.app.MockInsuranceViewModel.Companion.NORWEGIAN_HOME_CONTENTS
+import com.hedvig.app.MockInsuranceViewModel.Companion.NORWEGIAN_HOME_CONTENTS_AND_TRAVEL
+import com.hedvig.app.MockInsuranceViewModel.Companion.NORWEGIAN_TRAVEL
+import com.hedvig.app.MockInsuranceViewModel.Companion.SWEDISH_HOUSE
+import com.hedvig.app.R
+import com.hedvig.app.feature.insurance.ui.InsuranceViewModel
+import com.hedvig.app.feature.loggedin.ui.LoggedInActivity
+import com.hedvig.app.feature.loggedin.ui.LoggedInTabs
+import com.hedvig.app.feature.loggedin.ui.LoggedInViewModel
+import com.hedvig.app.feature.referrals.MockLoggedInViewModel
+import com.hedvig.app.genericDevelopmentAdapter
+import com.hedvig.app.insuranceModule
+import com.hedvig.app.loggedInModule
+import com.hedvig.app.testdata.dashboard.INSURANCE_DATA
+import com.hedvig.app.testdata.dashboard.INSURANCE_DATA_NO_RENEWAL
+import org.koin.android.viewmodel.dsl.viewModel
+import org.koin.core.context.loadKoinModules
+import org.koin.core.context.unloadKoinModules
+import org.koin.dsl.module
+
+
```

Comment:
Check if you need to re-apply `ktlint` on your project, and if save actions still works, to hammer out formatting quirks.